docs/source/operators/logging-events.md

```diff
@@ -0,0 +1,48 @@
+# Event logging
+
+Jupyter Server emits the following list of events (via [`jupyter_events`](https://jupyter-events.readthedocs.io/en/latest/))
+
+{{ jupyter_server_events }}
+
+## Collecting emitted events
+
+Configure the `EventLogger` to begin emitting events from Jupyter Server
+
+```python
+from logging import FileHandler
+
+event_handler = FileHandler("events.log")
+
+c.EventLogger.handlers = [event_handler]
```

Comment:
Does this work?  I believe when I tried this, I found I had to use `register_handler()`, otherwise the [`_logger` attribute doesn't get set](https://github.com/kevin-bates/jupyter_events/blob/181371f245c042e78c671225b8c257c015832bd7/jupyter_events/logger.py#L168).

Ah, is this because this is a config entry and the `EventLogger` instance does not yet exist, in which case its initializer will [do the right thing](https://github.com/jupyter/jupyter_events/blob/181371f245c042e78c671225b8c257c015832bd7/jupyter_events/logger.py#L113-L115)?  I had been trying to set the handlers list _after_ `EventLogger`'s instantiation.

Reply:
> Ah, is this because this is a config entry

Yeah, that's exactly right. This is a config syntax. If you do this after instantiation, the handlers trait does not update.